### PR TITLE
Ad integration fixes

### PR DESCRIPTION
**Pull request by spetrosi**

### CODE REVIEW

**Review by richm — APPROVED** [review]

File: tests/tests_configure_ha_cluster.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@
 - name: Verify the role on the primary node
   hosts: all
   vars:
-    __mssql_single_node_test: true
+    __mssql_single_node_test: "{{
```
Comment (richm) [inline tests/tests_configure_ha_cluster.yml:6]:
```suggestion
    __mssql_single_node_test: "{{ ansible_play_hosts_all | length <= 1 }}"
```
or `length == 1` if you don't care if the length is 0

Comment (spetrosi) [reply tests/tests_configure_ha_cluster.yml:6]: how `ansible_play_hosts_all` can be 0?

Comment (richm) [reply tests/tests_configure_ha_cluster.yml:6]: > how `ansible_play_hosts_all` can be 0?

I don't think it can be so `length == 1` is ok

**Comment by spetrosi** [comment]
[citest]

**Comment by spetrosi** [comment]
[citest]

**Pull request merged**